Python/Product/Debugger/Debugger/DebugEngine/Remote/PythonRemoteProcess.cs

```diff
@@ -136,9 +137,20 @@ private PythonRemoteProcess(int pid, Uri uri, PythonLanguageVersion langVer)
         }
 
         public static PythonProcess Attach(Uri uri, bool warnAboutAuthenticationErrors) {
+            string debugOptions = "";
+            if (uri.Query != null) {
+                // It is possible to have more than one opt=... in the URI - the debug engine will always supply one based on
+                // the options that it gets from VS, but the user can also explicitly specify it directly in the URI when doing
+                // ptvsd attach (this is currently the only way to enable some flags when attaching, e.g. Django debugging).
+                // ParseQueryString will automatically concat them all into a single value using commas.
+                var queryParts = HttpUtility.ParseQueryString(uri.Query);
+                debugOptions = queryParts[AD7Engine.DebugOptionsKey];
+            }
+
             var stream = Connect(uri, warnAboutAuthenticationErrors);
             try {
                 stream.Write(AttachCommandBytes);
+                stream.WriteString(debugOptions);

```

Comment:
This is writing an int as a string, correct? Because we put the options into the URI above as an int?